models/privacy.go

```diff
@@ -0,0 +1,7 @@
+package models
```

Comment:
OTT in my opinion, if `cfg.HasPrivateEndpoints` is true or false will suffice in any scenario within application; privacy.go not needed.